### ISSUE TITLE
PSAT AVR Types II and III - Initialisation

### ISSUE THREAD

**Issue by tinrabuzin** [itesla/ipsl#155]
Initialisation of the field excitation voltage is done by setting the vf0 parameter. Instead, it should be initialised by the vf0 pin from a generator.

**Comment by ahsanKTH** [comment]
Yes the same problem is with the Type I too. I can add an input connector for vf0 in AVR models so that it can be connected with the generator Vf0 output connector to initialize vf0 of AVR. 

Only problem would be finding the examples which used these AVR models and update the examples too.

**Comment by tinrabuzin** [comment]
@ahsanKTH Yes, we should do it in that way. Doing more work cannot be an excuse for not changing  it. :)

**Comment by ahsanKTH** [comment]
@tinrabuzin Yes I am in favor of doing it but I just wanted to mean that if we update the models we should update all the examples too.

**Comment by lvanfretti** [comment]
@tinrabuzin is this still a problem in OpenIPSL? If yes, I will move the post to our fork and close it here.
